config/validation.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+	"github.com/contiv/errored"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+
+	gojson "github.com/xeipuuv/gojsonschema"
+)
+
+// Combines array of errors into a single error
+func combineErrors(resultErrors []gojson.ResultError) error {
```

Comment:
gojsonschema package returns array of errors([]gojson.ResultError) that violated the validation rule. Like below:
backends.mount: backends.mount must be one of the following: "ceph", "nfs"
backends.mount: String length must be greater than or equal to 1

I'm combining all the errors into a single string and creating a error using errored.New()

I can use ```(*errored.Error).Combine``` but there will be stack trace for each one of them (Is it necessary? ) in debug mode. Also, list of errors can be huge based on  the JSON.

Please input your thoughts.